scripts/apps/authoring-react/authoring-angular-integration.tsx

```diff
@@ -184,6 +206,97 @@ function getInlineToolbarActions(
         return actions;
     };
 
+    const correctAction = (text: string = gettext('SEND CORRECTION'), tooltip?: string): ITopBarWidget<IArticle> => ({
```

Comment:
Identical strings have to be used for labels as in old authoring. e.g. "Send Correction". Otherwise existing translations couldn't be reused.